app/src/main/java/org/simple/clinic/bp/history/adapter/BloodPressureHistoryListItem.kt

```diff
@@ -27,6 +30,9 @@ sealed class BloodPressureHistoryListItem : ItemAdapter.Item<Event> {
                 showDivider = !isLastMeasurement
             )
           }
+          .let {
+            listOf(NewBpButton) + it
+          }
```

Comment:
Instead of doing this, I'd recommend extracting the history items to a separate variable for better readability.

```kotlin
val bpHistoryItems = measurements
  .mapIndexed { /* Generate measurement items */ }

return listOf(NewBpButton) + bpHistoryItems
```